aqueductcore/frontend/src/helper/auth/userScope.tsx

```diff
@@ -39,3 +39,24 @@ export function isUserAbleToDeleteExperiment(userInfo: UserInfo, createdBy: Expe
     return false
 }
 
+export function isUserAbleToCreateTask(userInfo: UserInfo) {
+    for (const item of userInfo.scopes) {
+        if (item === defined_scopes.JOB_CREATE) {
+            return true
+        }
+    }
+    return false
+}
+
+export function isUserAbleToCancelTask(userInfo: UserInfo, createdBy: ExperimentData['createdBy']) {
+    for (const item of userInfo.scopes) {
+        if (
+            item === defined_scopes.JOB_CANCEL_OWN ||
+            item === defined_scopes.JOB_CANCEL_ALL && userInfo.username === createdBy
+        ) {
```

Comment:
For better readability, we can add brackets

```suggestion
        if (
            item === defined_scopes.JOB_CANCEL_ALL ||
            (item === defined_scopes.JOB_CANCEL_OWN && userInfo.username === createdBy)
        ) {
```